python/dlisio/lis/load.py

```diff
@@ -89,7 +94,7 @@ def read_as_tapemark(f):
         # All delimiter records are indexed separately by index_records. If
         # applicable the raw record is extracted and correctly stored. In any
         # case the filehandle is closed before we continue indexing the file.
-        first = index.explicits()[0]
+        first = index.explicits()[0] if len(index.explicits()) else None
         if is_delimiter(first):
```

Comment:
So now it can create a file with empty index because is_delimiter(None) will return false?